src/openms/include/OpenMS/KERNEL/MSExperiment.h

```diff
@@ -551,6 +551,8 @@ namespace OpenMS
     /// returns true if any MS spectra of the specified level contain at least one peak with intensity of 0.0
     bool hasZeroIntensities(size_t ms_level) const;
 
+    /// do any of the spectra have a peptideID?
+    bool hasPeptideIdentifications() const;
```

Comment:
not part of this PR but imho IDs should not be stored in MSExperiment

Reply:
I agree!
But its used in many places...

Reply:
It is ugly enough so I opened an issue